src/test/oracle/OracleHelper.t.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright 2022 Spilsbury Holdings Ltd
+pragma solidity >=0.8.4;
+
+import {Test} from "forge-std/Test.sol";
+import {OracleHelper} from "../../libraries/OracleHelper.sol";
+import {IUniswapV3PoolImmutables} from "../../bridges/uniswapv3/interfaces/pool/IUniswapV3PoolImmutables.sol";
+import {IUniswapV3PoolDerivedState} from "../../bridges/uniswapv3/interfaces/pool/IUniswapV3PoolDerivedState.sol";
+import {TickMath} from "../../bridges/uniswapv3/libraries/TickMath.sol";
+import {FullMath} from "../../bridges/uniswapv3/libraries/FullMath.sol";
+
+contract OracleHelperTest is Test {
+    address[] private pools = [
+        0x88e6A0c2dDD26FEEb64F039a2c41296FcB3f5640, // USDC/WETH 500 bps
+        0x4e0924d3a751bE199C426d52fb1f2337fa96f736, // LUSD/USDC 500 bps
+        0xD1D5A4c0eA98971894772Dcd6D2f1dc71083C44E, // LQTY/WETH 3000 bps
+        0x4e68Ccd3E89f51C3074ca5072bbAC773960dFa36, // WETH/USDT 3000 bps
+        0x3416cF6C708Da44DB2624D63ea0AAef7113527C6 // USDC/USDT 100 bps
+    ];
+
+    function testSlippageToAmountOutMin(int16 _slippageBps, uint96 _amountIn) public {
+        vm.assume(_slippageBps >= 10 && _slippageBps <= 2000 && _amountIn > 1e15);
+        emit log_int(_slippageBps);
```

Comment:
Should use `bound` from `Test` instead of `vm.assume` also remove the `log_int`.

Reply:
Just updated the code with bound(,,,). It resulted in some ugly type conversion though 😏